addOns/openapi/src/test/resources/org/zaproxy/zap/extension/openapi/v3/petstore_reduced_intl_phone.yaml

```diff
@@ -0,0 +1,160 @@
+openapi: "3.0.0"
```

Comment:
I'd suggest trim to the absolutely minimum required for the test.

Reply:
I'll be honest, I don't actually "know". I tried to trim it down to what seemed reasonable but I don't know the add-on or OpenAPI spec well enough.

Reply:
Ok if I push some changes and you then fix up?

Reply:
100% 

Thank you!

Reply:
Done.